ethexe/ethereum/src/lib.rs

```diff
@@ -297,7 +297,9 @@ impl<T: Transport + Clone, N: Network> TryGetReceipt<T, N> for PendingTransactio
             Err(err) => err,
         };
 
-        for _ in 0..3 {
+        log::trace!("Failed to get transaction receipt for {tx_hash}. Retrying...");
+        for n in 0..25 {
```

Comment:
it's ok for tests, but bad for production

Reply:
not actual